content/realtime/channels.textile

```diff
@@ -236,6 +236,149 @@ channel.subscribe { message in
 }
 ```
 
+h4(#rewind). Subscribing to a channel with rewind option enabled
+
+If you wish to both subscribe to a channel, as well as obtain some messages from said channel's "history":/realtime/history seamlessly, you can make use of the "rewind":/realtime/channel-params feature. This will result in the specified number of messages from "history":/realtime/history being returned as part of the subscription process.
```

Comment:
Suggest changing the first sentence to:

'It is possible to subscribe to a channel and obtain messages from that channel's "history":/realtime history with a single API request. To do so you would use the "rewind":/realtime/channel-params feature.'